lms/services/assignment.py

```diff
@@ -316,6 +328,22 @@ def get_courses_assignments_count(self, **kwargs) -> dict[int, int]:
 
         return {x.course_id: x.count for x in self._db.execute(query)}  # type: ignore
 
+    def _update_auto_grading_config(
```

Comment:
This could become a `update_from_dict` or similar directly in AutoGradingConfig if we end up doing the same in multiple locations.